test/loader.ts

```diff
@@ -0,0 +1,85 @@
+/*
+   Batch loader
+ */
+
+// @ts-ignore
+import DataLoader from 'dataloader';
+import { DefaultSchema } from './schemas';
+import { Client, dynamoExecutedCommandsTracer, Table } from './utils/init';
+
+const table = new Table({
+  name: 'BatchTest',
```

Comment:
Small nit. Can you format all your code / files with TS=4 spaces

Reply:
I'm just picking at little things to tidy up before merging.

I'd like to keep this under a prototype status in the main branch. i.e. Not document yet while you battle harden it in practice and others get to play with it. Once documented, it is set in stone.

Reply:
done the reformatting, ready to go I suppose :)